posh-svn.psd1

```diff
@@ -0,0 +1,56 @@
+@{
+
+# Script module or binary module file associated with this manifest.
+ModuleToProcess = 'posh-svn.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.7.301'
```

Comment:
Where does this originate from?

Reply:
Sorry... I put in 0.7 to match posh-git on my local and the higher number was to avoid it being overwritten if I updated the module for whatever reason.  I'll change it to whatever makes the most sense.